.github/workflows/build.yml

```diff
@@ -43,3 +43,8 @@ jobs:
         uses: GabrielBB/xvfb-action@v1.5
         with:
           run: npm test
+      
+      - name: Extension packaging
+        run: |
+          sudo npm install -g vsce
+          vsce package
```

Comment:
please publish the extension to the gh packages

Reply:
^^

Reply:
I will do this once releases are set up

Reply:
gh packages doesn't support publishing of vsix applications